app/src/main/java/com/rkm/tasky/feature/common/ScreenType.kt

```diff
@@ -0,0 +1,18 @@
+package com.rkm.tasky.feature.common
+
+import androidx.annotation.StringRes
+import com.rkm.tasky.R
+
+enum class ScreenType() {
+    EVENT,
+    TASK,
+    REMINDER
+}
+
+fun ScreenType.toUiString(): Int {
+    return when(this) {
+        ScreenType.EVENT -> R.string.event
+        ScreenType.TASK -> R.string.task
+        ScreenType.REMINDER -> R.string.reminder
+    }
+}
```

Comment:
Could it make sense to call this `AgendaItemType` instead, so it's more universally applicable? In the end, the screen is constructed based on the type of the item and not the item based on the type of screen 😄